Arreglar compilación al activar EMAC sun50i en la configuración de U-Boot

El controlador EMAC sun50i asume que están definidas algunas constantes específicas
de un modelo de GMAC que no coincide con el de la Pine. Al no manejar su ausencia de definición
apropiadamente, esto genera un error de compilación.

Este parche quita el código responsable de dar soporte a un modelo de GMAC no relacionado
con el de la Pine, y permite que U-Boot compile.

Signed-off-by: AlexTMjugador <AlexTMjugador@users.noreply.github.com>
---
diff -pur uboot.orig/drivers/net/sun8i_emac.c uboot/drivers/net/sun8i_emac.c
--- uboot.orig/drivers/net/sun8i_emac.c	2019-08-25 13:11:10.363845695 +0200
+++ uboot/drivers/net/sun8i_emac.c	2019-08-25 13:12:21.611843422 +0200
@@ -291,17 +291,6 @@ static int sun8i_emac_set_syscon(struct
 	int ret;
 	u32 reg;
 
-	if (priv->variant == R40_GMAC) {
-		/* Select RGMII for R40 */
-		reg = readl(priv->sysctl_reg + 0x164);
-		reg |= CCM_GMAC_CTRL_TX_CLK_SRC_INT_RGMII |
-		       CCM_GMAC_CTRL_GPIT_RGMII |
-		       CCM_GMAC_CTRL_TX_CLK_DELAY(CONFIG_GMAC_TX_DELAY);
-
-		writel(reg, priv->sysctl_reg + 0x164);
-		return 0;
-	}
-
 	reg = readl(priv->sysctl_reg + 0x30);
 
 	if (priv->variant == H3_EMAC) {
